waiter/src/waiter/service_description.clj

```diff
@@ -205,8 +219,21 @@
   "If the provided allowed-params was invalid, attempt to generate a friendly error message.
    Return nil for unknown error."
   [issue]
-  (when (get issue "allowed-params")
-    "allowed-params must be a vector with non-empty strings."))
+  (when-let [allowed-params-issue (get issue "allowed-params")]
+    (let [length-violation (some #(= "schema.core.Constrained" (some-> % .schema .getClass .getCanonicalName))
```

Comment:
Can we use `instance?`

Reply:
Do we need the `some->`?

Reply:
Changed to use `instance?` and removed the `some->`